### ISSUE TITLE
Can't create album when Google Photos has no photos.

### ISSUE THREAD

**Issue by ManuCart** [3846masa/upload-gphotos#228]
Hi,
I recently updated your script and now I have this error
````
ERROR Album "test album" is not found.
ERROR TypeError: Cannot read property 'map' of null
    at /usr/local/lib/node_modules/upload-gphotos/lib/index.js:397:35
    at Generator.next (<anonymous>)
    at onFulfilled (/usr/local/lib/node_modules/upload-gphotos/node_modules/co/index.js:65:19)
````
It's seems your script try to create album name before uploading photos.
What's wrong?

**Comment by 3846masa** [comment]
Google Photos requires least one photo which already uploaded to create album.
I think your account have no photo in Google Photos, so please upload any photo via web before using cli.

Note: I'll try to fix this bug.

**Comment by ManuCart** [comment]
You right, I remove every photos and every albums to do a new fresh version.
I was thinking about that before posting this error and put a photo by the html interface.
But no way, I think it's an other problem, maybe I block my account because I remove 35000 photos and 1500 albums (with auto mouse click).
I try too with an other account and it's work.
I think, I need to wait couple of days and try again.
We wil see, thanks for your help

**Comment by 3846masa** [comment]
Fixed and released as v1.4.4.
